src/client/js/partials/breaches.js

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const breachesPartial = document.querySelector("[data-partial='breaches']")
+const chartColors = ['#321C64', '#AB71FF', '#952BB9', '#D74CF0', '#9e9e9e']
```

Comment:
`#AB71FF` and `#D74CF0` don't provide enough context at this font size, making them hard to read for people with e.g. lots of backlight or low-quality displays. Perhaps we could ask Design for darker colours?

![image](https://user-images.githubusercontent.com/4251/220894303-f8e1ff21-9db5-4878-9061-919d46a095be.png)

![image](https://user-images.githubusercontent.com/4251/220894335-0fd992f4-13e7-4848-b912-54bf24e6dcb9.png)

Reply:
Was thinking the exact thing during dev – I'll flag with design 👍

Reply:
Fixed via https://github.com/mozilla/blurts-server/pull/2862/commits/fb71affa7c554763a84e804f597cd550e387ed9c - now using dark gray for text.